Analysis/Tasks/PWGHF/HFD0CandidateSelector.cxx

```diff
@@ -113,35 +71,35 @@ struct HFD0CandidateSelector {
   bool selectionTopol(const T& hfCandProng2)
   {
     auto candpT = hfCandProng2.pt();
-    int pTBin = getpTBin(candpT);
+    auto pTBin = findPTBin(ptBins, candpT);
     if (pTBin == -1) {
       return false;
     }
 
     if (candpT < d_pTCandMin || candpT >= d_pTCandMax) {
       return false; //check that the candidate pT is within the analysis range
     }
-    if (hfCandProng2.impactParameterProduct() > cuts[pTBin][7]) {
+    if (hfCandProng2.impactParameterProduct() > cuts->get(pTBin, "d0d0")) {
       return false; //product of daughter impact parameters
     }
-    if (hfCandProng2.cpa() < cuts[pTBin][8]) {
+    if (hfCandProng2.cpa() < cuts->get(pTBin, "cos pointing angle")) {
```

Comment:
I wonder whether allowing the use of spaces in the cut name definition is a good idea. @aalkin don't you think it could trigger errors?

Reply:
We can also have an `enum` defined, but string are more generic.